src/main/resources/schemas/devfile.json

```diff
@@ -144,7 +141,7 @@
             ],
             "properties": {
               "commandLine": {
-                "description": "The actual command-line string\n\nSpecial variables that can be used:\n\n - `$PROJECTS_ROOT`: A path where projects sources are mounted as defined by container component's sourceMapping.\n\n - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/\u003cproject-name\u003e). If there are multiple projects, this will point to the directory of the first one.",
+                "description": "The actual command-line string\n\nSpecial variables that can be used:\n\n - `$PROJECTS_ROOT`: A path where projects sources are mounted as defined by container component's sourceMapping.\n\n - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/<project-name>). If there are multiple projects, this will point to the directory of the first one.",
```

Comment:
original json uses \u003c, can you also uses it ? 
https://github.com/devfile/api/blob/a6ec0a38307b63a29fad2eea945cc69bee97a683/schemas/latest/devfile.json#L144

Reply:
Uh. Sorry, I don't know why this happened during the copy. Updated it